src/newton.jl

```diff
@@ -134,7 +82,7 @@ function newton{T}(f::Function, f_prime::Function, x::Interval{T}, level::Int=0;
     debug && @show(deriv)
 
     if !(z in deriv)
-        Nx = N(f, x, deriv)
+        Nx = 𝒩(f, x, deriv)
```

Comment:
I think this should read `𝒩(f, deriv, x)` and a method of `𝒩` can be safely deleted, as noted above.